app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,29 @@
 <resources>
     <string name="app_name">Chapter2Programs</string>
+    <string name="tvName"> Enter Your Name </string>
+    <string name="etEmail"> Enter Your Email </string>
+    <string name="hint_text_details"> "Feedback Details..." </string>
+    <string name="checkbox_text_response"> "Would you like an email response?" </string>
+    <string name="button_text_send"> Submit </string>
+    <string name="tvInstruction">Enter feedback details to send to the developers </string>
+    <string name="ch2">Chapter 2</string>
+    <string name="ch3">Chapter 3</string>
+    <string name="linear">Linear Layout</string>
+    <string name="constraint">Constraint Layout</string>
+    <string name="frame">Frame Layout</string>
+    <string name="table">Table Layout</string>
+    <string name="ui">Beauty and the Beast UI</string>
```

Comment:
naming is not proper for ui. title_activity_android_layout is proper naming, change other names according to this